.vscode/launch.json

```diff
@@ -462,13 +462,11 @@
                 "server",
                 "start",
                 "--mode",
-                "consolidated",
-                "--namespace",
-                "storage"
+                "consolidated"
             ],
             "cwd": "${workspaceFolder}",
             "console": "integratedTerminal",
-            "stopAtEntry": true,
+            "stopAtEntry": false,
```

Comment:
What's the difference between using `true` and `false` here? Wouldn't we want to have "consolidated" mode trump everything else if enabled?

Reply:
`stopAtEntry` means debugger will stop right when it enters the program, which is not needed, as I usually have set my breakpoints before I start the debugger. It is a debugger feature, which has nothing to do with our product.